minecraft_servers/firecraft/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+    "server_name": "firecraft",
+    "nice_name": "FireCraft",
+    "direct_ip": "firecraft.dk",
+    "server_wildcards": null,
```

Comment:
`null` is not allowed here. If you do not have any wildcards, just remove, as it is not a required key.